api/framework/announcements/bundle.md

```diff
@@ -0,0 +1,31 @@
+# announcements
+
+Bundle for displaying announcements made with admin-announcements.
+
+## Description
+
+The bundle displays announcements made with admin-announcements bundle. The announcements are displayed in chronological order. Announcements can also show as a pop-up, they can be disabled with "Don't show this again" after seen.
+
+## TODO
+
+* Filtering options
+* Ordering options
+* Create proper error logging/messages for ajax calls
+* Tests
+
+## Bundle configuration
+
+Frontend has no configuration, but the bundle is usually configured to be shown only to admin users. Configuration is done in oskari-server using oskari-ext.properties.
+
+## Dependencies
+
+<table class="table">
+  <tr>
+    <th> Dependency </th><th> Linked from </th><th> Purpose </th>
+  </tr>
+  <tr>
+    <td> [Oskari admin-announcements](/documentation/bundles/framework/admin-announcements) </td>
+    <td> Expects to be present in application setup </td>
+    <td> Admin-announcements bundle to create the announcements</td>
+  </tr>
+</table>
```

Comment:
I hope the admin isn't a dependency for the announcements. If you want to indicate that there is another bundle that relates to this maybe you can just tell that on the desciption with "Announcements have an admin UI implemented as bundle [bundle]"?